src/core/ext/filters/client_channel/lb_policy/weighted_round_robin/weighted_round_robin.cc

```diff
@@ -445,7 +445,7 @@ float WeightedRoundRobin::AddressWeight::GetWeight(
     gpr_log(GPR_INFO,
             "[WRR %p] subchannel %s: getting weight: now=%s "
             "weight_expiration_period=%s blackout_period=%s "
-            "last_update_time_=%s non_empty_since_=%s weight_=%f",
+            "last_update_time=%s non_empty_since=%s weight=%f",
```

Comment:
Something went wrong during `merge master` which gave me a "local copy is outdated" error, which I had to force-push. I think this change was related. Fixed and ack on force-push, I think in this case I can `merge origin` to avoid force-pushing later.